.github/workflows/ci.yml

```diff
@@ -55,24 +104,29 @@ jobs:
         if: github.event_name == 'pull_request_target'
       - uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.version }}
+          python-version: ${{ matrix.python }}
           architecture: 'x64'
       - uses: actions/cache@v3
         with:
           path: |
             ~/.cache/pip
             .nox
-          key: tests-os-${{ runner.os }}-python-${{ matrix.python }}-airflow-${{ matrix.airflow }}-deps-${{ hashFiles('pyproject.toml') }}-version-${{ hashFiles('src/astro_databricks/__init__.py') }}
+          key: unit-tests-os-${{ runner.os }}-python-${{ matrix.python }}-airflow-${{ matrix.airflow }}-deps-${{ hashFiles('pyproject.toml') }}-version-${{ hashFiles('src/astro_databricks/__init__.py') }}
       - run: sqlite3 /tmp/sqlite_default.db "VACUUM;"
       - run: pip3 install nox
-      - run: nox -s "test-${{ matrix.version }}(airflow='${{ matrix.airflow }}')" -- tests/ --ignore "tests/example_dags" --cov=src --cov-report=xml --cov-branch
+      - run: nox -s "test-${{ matrix.python }}(airflow='${{ matrix.airflow }}')" -- tests/ --ignore "tests/test_example_dags.py" --cov=src --cov-report=xml --cov-branch
       - name: Upload coverage
         uses: actions/upload-artifact@v2
         with:
           name: coverage-${{ matrix.python }}-${{ matrix.airflow }}
           path: ./.coverage
 
-  Run-example-dag-tests-Airflow-2-5:
+  Run-example-dag-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [ '3.8', '3.9']
```

Comment:
At the moments, in our `pyproject.toml`, we say we also support 3.7 and 3.10 - should we add them here as well?

Reply:
Trying a run for the same.

Reply:
It ran fine!